package.json

```diff
@@ -16,13 +16,13 @@
   },
   "private": true,
   "dependencies": {
-    "@aerogear/app": "2.6.3",
```

Comment:
2.6 to 2.8? Why?

Reply:
A couple of releases of the JavaScript SDKs have been made and the showcase hasn't been updated accordingly.

Reply:
Wow. We should not let this happen